### ISSUE TITLE
ebay-listbox-button: aria-labelledby not adding id on inner text element

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1724]
# Bug Report

## Description

Here I set prefixId to “foo” and it correctly adds the aria-labelledby="foo c1-label" to the button. However, c1-label does not exist (it should be dynamically added to the span element around the inner button text).

![Screen Shot 2022-08-12 at 12 24 19 PM](https://user-images.githubusercontent.com/38065/184430578-f9bf4e9a-db0a-41b9-9fed-861f2c4a4cbd.png)

Compare this to ebay-menu-button:

![Screen Shot 2022-08-12 at 12 29 56 PM](https://user-images.githubusercontent.com/38065/184430619-fb6f47c9-d808-4931-a9f3-9cbfd244d866.png)

## Workaround

None

**Comment by agliga** [comment]
Yes, it adds the ID but only in certain cases. 
If there is a prefix label or floating label, then it does not add it. This is how we set it up in the code
If floating label and prefix label are not set, then it does add the ID

**Comment by agliga** [comment]
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-listbox-button/index.marko#L50-L63

**Comment by ianmcburnie** [comment]
Ah yes, thanks, I'd forgotten that prefixLabel was also being set. Here it is with prefixLabel removed:

![Screen Shot 2022-08-12 at 1 57 27 PM](https://user-images.githubusercontent.com/38065/184444064-f6b7b762-d18f-46d2-917d-1a09c72e0077.png)

Closing ticket.
